go.mod

```diff
@@ -223,7 +223,7 @@ replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
 	// Use cosmos-sdk fork with staking transfer events, and custom tally handler support
 	github.com/cosmos/cosmos-sdk => github.com/kava-labs/cosmos-sdk v0.47.10-kava.2
-	github.com/ethereum/go-ethereum => github.com/Kava-Labs/go-ethereum v1.10.27-0.20240308170502-da7973e5eee0
+	github.com/ethereum/go-ethereum => github.com/Kava-Labs/go-ethereum v1.10.27-0.20240513233504-6e038346780b
```

Comment:
No, it relies on latest commit of `kava/release/1.10/` which is our main branch in geth fork.
So basically I just updated geth dependency, because I needed `NewEVMWithEnabledPrecompiles` which we recently merged into geth fork.